rest-api/offline/main.py

```diff
@@ -153,6 +157,12 @@ def _build_pipeline_app():
       view_func=export_tables,
       methods=['POST'])
 
+  offline_app.add_url_rule(
+    PREFIX + 'RotateKeys',
```

Comment:
You could rename the endpoint to "DeleteServiceAccountKeys" to match.

You might also want to put "old" in there like "DeleteOldServiceAccountKeys"... we want to make it clear we're not deleting everything. :)

Reply:
done